modules/qtwidgets/include/modules/qtwidgets/inviwoqtutils.h

```diff
@@ -176,12 +176,19 @@ IVW_MODULE_QTWIDGETS_API QMenu* getMenu(std::string menuName, bool createIfNotFo
 IVW_MODULE_QTWIDGETS_API QImage layerToQImage(const Layer& layer);
 
 /*
- * \brief save the given QImage \p image as png in a base64-encoded string
+ * \brief save the given QImage \p image as base64-encoded string using the image file format \p
+ * format and image \p quality.
  *
  * @param image    image to be encoded
- * @return base64 string of the corresponding png image
+ * @param format   image file format
+ * @param quality  image quality [0,100] (0 - small compressed, 100 - large uncompressed),
+ *                 -1 uses default settings
+ * @return base64 string of the corresponding image
+ *
+ * \see QImage::save()
  */
-IVW_MODULE_QTWIDGETS_API std::string toBase64(const QImage& image);
+IVW_MODULE_QTWIDGETS_API std::string toBase64(const QImage& image,
+                                              const std::string& format = "PNG", int quality = -1);
```

Comment:
Should format be jpeg by default?

Reply:
Don't think so. We use the base64 encoding also for the port inspectors. There we should definitely use some lossless format :-)